### PR TITLE
Adds Gold Paint

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
Adds gold paint, silk shirt and narcotics not included.
drip guns

## Why It's Good For The Game
drip

## Changelog
:cl:
add: Added gold paint and scaramanga disk
code: reformatted some contraband in vendors
/:cl:

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1452,7 +1452,10 @@
 					/obj/item/weapon/computer_hardware/hard_drive/portable/design/nonlethal_ammo = 10,
 					/obj/item/weapon/electronics/circuitboard/autolathe = 3,
 					/obj/item/weapon/electronics/circuitboard/vending = 10)
-	contraband = list(/obj/item/weapon/computer_hardware/hard_drive/portable/design/lethal_ammo = 3, /obj/item/weapon/electronics/circuitboard/autolathe_disk_cloner = 3)
+	contraband = list(
+			/obj/item/weapon/computer_hardware/hard_drive/portable/design/lethal_ammo = 3,
+			/obj/item/weapon/electronics/circuitboard/autolathe_disk_cloner = 3,
```
Comment (Firefox13) [inline code/game/machinery/vending.dm:1457]:
```suggestion
			/obj/item/weapon/electronics/circuitboard/autolathe_disk_cloner = 3
```

File: code/game/objects/items/weapons/design_disks/autolathe_disks.dm [hunk 2]
```diff
@@ -79,6 +79,18 @@
 		/datum/design/autolathe/cell/medium/high,
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/scaramanga
+	disk_name = "\"Scaramanga\" gold set"
+	icon_state = "onestar"
+	rarity_value = 10
```
Comment (Firefox13) [inline code/game/objects/items/weapons/design_disks/autolathe_disks.dm:85]: bruh

Comment (zombiemimic) [reply code/game/objects/items/weapons/design_disks/autolathe_disks.dm:85]: what?

**Comment by Alliostra** [comment]
you know that you can already use the paint chemicals to color things, right? (though they don't look as good), might be a better idea to build upon that instead of having 2 different ways of painting things, maybe?

**Pull request merged**